src/popup/views/CreatePassword/index.tsx

```diff
@@ -58,7 +68,6 @@ const CreatePassword = () => {
       header="Create a password"
       subheader="Min 10 characters"
       icon={EMOJI.see_no_evil}
-      goBack={() => history.push({ pathname: "/" })}
```

Comment:
ah sorry - i know there isn't a back button in the designs, but Bruno added it in the wireframe and we didn't update design. let's leave this in:

https://www.figma.com/file/uIoeq72R2XjiWjGyVGiPG6/Lyra-wireframes?node-id=0%3A1

Reply:
Added back